CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Change Log
 
-## Unreleased
-
+## Dangerzone 0.4.0
+
+- Platform support: Re-add Fedora 37 support
+- Platform support: Add Debian Bookworm (12) support
```

Comment:
We can mention here this issue: https://github.com/freedomofpress/dangerzone/issues/172